utilities/cli/holohub.py

```diff
@@ -314,6 +314,11 @@ def _create_parser(self) -> argparse.ArgumentParser:
         setup.add_argument(
             "--dryrun", action="store_true", help="Print commands without executing them"
         )
+        setup.add_argument(
+            "--scripts",
```

Comment:
Since `./holohub setup` doesn't have any other argument. Would it be possible to change the api from `--script coverage` to `--coverage`?
It would provide a more natural interface to the users.

One way that it can be implemnted is to first parse the HOLOHUB_SETUP_SCRIPTS_DIR directory and find the scripts. Then for each script name create add an argparse argument here.